x-pack/plugins/lens/public/indexpattern_datasource/to_expression.ts

```diff
@@ -121,7 +123,7 @@ function getExpressionForLayer(
       return null;
     }
     const idMap = esAggEntries.reduce((currentIdMap, [colId, column], index) => {
-      const esAggsId = `col-${index}-${colId}`;
+      const esAggsId = `col-${index}-${index}`;
```

Comment:
hmmm, why do we have `${index}-${index}` here?

Reply:
This is how tabify (which is part of the `esaggs` function rolls): https://github.com/elastic/kibana/blob/4584a8b570402aa07832cf3e5b520e5d2cfa7166/src/plugins/data/common/search/tabify/get_columns.ts#L23